Examples/DicomSeriesReadModifyWrite/DicomSeriesReadModifySeriesWrite.py

```diff
@@ -79,7 +79,7 @@
     "0010|0010",  # Patient Name
     "0010|0020",  # Patient ID
     "0010|0030",  # Patient Birth Date
-    "0020|000D",  # Study Instance UID, for machine consumption
+    "0020|000d",  # Study Instance UID, for machine consumption
```

Comment:
On line 105 we are using `tolower`, which I guess is not technically needed any more.

Should the "Note" down below be promoted to the Documentation.rst?

Reply:
Agreed on both counts. Will remove the `tolower` and *copy* the note to the Documentation.rst. Hopefully if it appears as is in enough places folks will be aware of the case sensitivity issue.

Reply:
Maybe start the note with "Alway write DICOM tags in lower case hexadecimals."

Reply:
Done and force pushed. Please merge once passes testing.